x-pack/plugins/enterprise_search/public/applications/app_search/components/not_found/not_found.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { APP_SEARCH_PLUGIN } from '../../../../../common/constants';
+import { PageTemplateProps } from '../../../shared/layout';
+import { NotFoundPrompt } from '../../../shared/not_found';
+import { AppSearchLogo } from '../../../shared/not_found/logos';
```

Comment:
Just curious... any reason we don't use the EUI version of our logo? For that matter, where else in the app are we using the product logos. Wondering why we don't use the Enterprise Search logo like the loading spinner

Reply:
> any reason we don't use the EUI version of our logo?

Last I was told during the initial plugin setup, EUI was eventually getting rid of `logoAppSearch` and `logoWorkplaceSearch` in favor of solution only logos 🤷 We can double-check again in the design or EUI channels!

> Wondering why we don't use the Enterprise Search logo like the loading spinner

Hmm ya maybe we just should lol. I think originally I added 404 screens during the project MVP and I was still very much in a "migrate it as closely as possible!" mode whereas now I'm clearly much wiser and try to streamline to EUI first. 🤓 

I'll leave it up to y'all, I've got no skin in the game either way - maybe we should ping Davey, Casey, or Jonas?

Reply:
Something for product/design to decide IMO

Reply:
👍 I'm pinging them now on Slack. If it's an easy sign off on switching to base EUI icons/logos, I'll probably do it in this PR. If I can't get a decision in like 3 hours I'll just merge this PR as-is

Reply:
Just to tie off this convo, Davey was +1 to switching to EUI's `logoEnterpriseSearch`: https://github.com/elastic/kibana/pull/103198/commits/ff5f2d9c1e985b1b1302025e812a53b791067e8b